app/src/main/java/org/sopt/sample/HomeActivity.kt

```diff
@@ -0,0 +1,18 @@
+package org.sopt.sample
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import org.sopt.sample.databinding.ActivityHomeBinding
+
+class HomeActivity: AppCompatActivity() {
+    private lateinit var binding: ActivityHomeBinding
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityHomeBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        val intent = intent
+        binding.idOutput.text = "ID: ".plus(intent.getStringExtra("id"))
+        binding.mbtiOutput.text = "MBTI: ".plus(intent.getStringExtra("mbti"))
```

Comment:
```suggestion
        binding.idOutput.text = "ID: ${intent.getStringExtra("id")}"
        binding.mbtiOutput.text = "MBTI: ${intent.getStringExtra("mbti")}"
```

이렇게 문자열 안에서 변수 값을 직접 사용할 수 있어요